docs/changelog.md

```diff
@@ -1,3 +1,11 @@
+## 0.17.1 (Feb 9, 2021)
+
+### Added
+
+* Add handler option for bubbles and keyboard buttons (`handler`). If `client`,
+  should be processed only by client
```

Comment:
and message will not come to bot. If \`bot\` message will send by client to bot.